src/cryptography/x509/base.py

```diff
@@ -73,7 +74,7 @@ class Version(Enum):
 
 
 class InvalidVersion(Exception):
-    def __init__(self, msg, parsed_version):
+    def __init__(self, msg: str, parsed_version: typing.Any):
```

Comment:
Is this really an `Any`?  Looks like it should be an `int`.

Reply:
The exception is only raised in one place (`src/cryptography/hazmat/backends/openssl/x509.py`), and there it's the return value of an internal function `X509_get_version()`.  The exception itself does not do anything with the value.

So I presume it's de facto always an int, and I've adjusted the code accordingly.